### PR TITLE
Add exlude warning, not include warning to total error

### PR DESCRIPTION
**Pull request by dimasciput**

### CODE REVIEW

File: flask_project/campaign_manager/insights_functions/feature_attribute_completeness.py [hunk 1]
```diff
@@ -119,9 +119,14 @@ def check_feature_completeness(
                         warning_message.append(warning)
 
         feature_data['error'] = 'False'
+        feature_data['warning'] = 'False'
+
         if warning_message or error_message:
```
Comment (cchristelis) [inline flask_project/campaign_manager/insights_functions/feature_attribute_completeness.py]: We are still generating errors based on warnings here. Shouldn't this just check for error_messages?

File: flask_project/campaign_manager/templates/campaign_widget/ui/feature_completeness.html [hunk 2]
```diff
@@ -175,6 +175,12 @@ <h3 class="progress-bar-indicator">{{ data.complete }}/{{ data.total }} ({{ data
         var _featureCompleteness = [];
         {% if row['error'] == 'True' %}
```
Comment (cchristelis) [inline flask_project/campaign_manager/templates/campaign_widget/ui/feature_completeness.html]: Then rather have an `if error == True or warning == True` here

**Comment by ann26** [comment]
In local: this PR already exclude warning on completeness
screenshot from 2017-11-06 11-18-42

**Comment by meomancer** [comment]
hi @ann26 
it seems that this PR is duplicated with your PR
https://github.com/hotosm/field-campaigner/pull/386
and it is merged

could you make sure it is ok if this PR closed?

**Comment by ann26** [comment]
yes @meomancer this is duplicate

**Pull request closed without merging**